src/openMVG/dataio/CMakeLists.txt

```diff
@@ -26,4 +26,4 @@ if(OpenMVG_USE_ALEMBIC)
 endif(OpenMVG_USE_ALEMBIC)
 install(TARGETS openMVG_dataio DESTINATION lib/ EXPORT openMVG-targets)
 
-#UNIT_TEST(openMVG dataio "openMVG_dataio")
\ No newline at end of file
+UNIT_TEST(openMVG AlembicImporter "openMVG_dataio")
```

Comment:
Also lines 12 and 13 should be uncommented, otherwise the cpp test file is added to the library.